### PR TITLE
SCA ignores for open policies

### PR DESCRIPTION
**Pull request by davidkelliott**
These are mostly for policies which are directly attached to the resource or similar and there for resources needs to be *.  They have been ignored for tfsec before but this is a new check with checkov now doing the same thing.  Also a couple of other ignores which have been verified as ok.

### CODE REVIEW

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 1]
```diff
@@ -96,6 +96,7 @@ data "aws_iam_policy_document" "developer_additional" {
   #checkov:skip=CKV_AWS_109
   #checkov:skip=CKV_AWS_111
   #checkov:skip=CKV_AWS_110
+  #checkov:skip=CKV_AWS_356
```
Comment (ewastempel) [inline terraform/environments/bootstrap/delegate-access/policies.tf:99]: can we have a reasoning why we are suppressing this message, please? like in the previous file:
`# checkov:skip=CKV_AWS_356: "Cannot restrict by KMS alias so leaving open"`

Comment (davidkelliott) [inline terraform/environments/bootstrap/delegate-access/policies.tf:99]:
```suggestion
  #checkov:skip=CKV_AWS_356: Needs to access multiple resources
```

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 2]
```diff
@@ -224,6 +225,7 @@ data "aws_iam_policy_document" "data_engineering_additional" {
...
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/delegate-access/policies.tf:228]:
```suggestion
  #checkov:skip=CKV_AWS_356: Needs to access multiple resources
```

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 3]
```diff
@@ -305,6 +307,7 @@ data "aws_iam_policy_document" "sandbox_additional" {
   #checkov:skip=CKV_AWS_109
   #checkov:skip=CKV_AWS_110
   #checkov:skip=CKV2_AWS_40
+  #checkov:skip=CKV_AWS_356
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/delegate-access/policies.tf:310]:
```suggestion
  #checkov:skip=CKV_AWS_356: Needs to access multiple resources
```

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 4]
```diff
@@ -485,6 +489,7 @@ data "aws_iam_policy_document" "instance-management-document" {
   #checkov:skip=CKV_AWS_109
   #checkov:skip=CKV_AWS_111
   #checkov:skip=CKV_AWS_110
+  #checkov:skip=CKV_AWS_356
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/delegate-access/policies.tf:492]:
```suggestion
  #checkov:skip=CKV_AWS_356: Needs to access multiple resources
```

File: terraform/environments/bootstrap/member-bootstrap/iam.tf [hunk 5]
```diff
@@ -22,6 +22,7 @@ data "aws_iam_policy_document" "member-access" {
     #checkov:skip=CKV_AWS_109
     #checkov:skip=CKV_AWS_110
     #checkov:skip=CKV2_AWS_40
+    #checkov:skip=CKV2_AWS_356
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/member-bootstrap/iam.tf:25]:
```suggestion
    #checkov:skip=CKV2_AWS_356: Needs to access multiple resources
```

File: terraform/environments/bootstrap/member-bootstrap/iam.tf [hunk 6]
```diff
@@ -255,6 +256,7 @@ data "aws_iam_policy_document" "member-access-us-east" {
...
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/member-bootstrap/iam.tf:259]:
```suggestion
    #checkov:skip=CKV2_AWS_356: Needs to access multiple resources
```

File: terraform/environments/ccms-ebs/iam.tf [hunk 7]
```diff
@@ -18,6 +18,7 @@ resource "aws_iam_user_policy" "email_policy" {
   policy = data.aws_iam_policy_document.email.json
 }
 
+#tfsec:ignore:aws-iam-no-policy-wildcards
```
Comment (davidkelliott) [inline terraform/environments/ccms-ebs/iam.tf:21]:
```suggestion
# Following AWS recommended policy
#tfsec:ignore:aws-iam-no-policy-wildcards
```

**Review by ewastempel — APPROVED** [review]
LGTM

**Pull request merged**